forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -21,15 +21,22 @@ import com.joinforage.forage.android.ui.ForageConfig
 import java.util.UUID
 
 /**
- * A class implementation of ForageSDKInterface
+ * An instance of the ForageSDK
  */
 class ForageSDK : ForageSDKInterface {
 
+    /**
+     * Retrieves the ForageConfig for a given ForageElement, or throws an exception if the ForageConfig is not set.
+     *
+     * @param element A ForageElement instance
+     * @return The ForageConfig associated with the ForageElement
+     * @throws ForageConfigNotSetException If the ForageConfig is not set for the ForageElement
+     */
     private fun <T : ElementState> _getForageConfigOrThrow(element: AbstractForageElement<T>): ForageConfig {
```

Comment:
This method is private. I think it's fine to leave it in the PR, but most of the KDoc comments should be for public-only code.